elmui/src/Main.elm

```diff
@@ -0,0 +1,62 @@
+port module Main exposing (..)
+
+import Browser
+import Html exposing (Html, Attribute, button, div, input, text, textarea)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onInput, onClick)
+
+
+
+-- MAIN
+
+
+main =
+  Browser.element { init = init, subscriptions = subs, update = update, view = view  }
+
+-- MODEL
+
+type alias Model =
+  { code : String,
+    result: String
+  }
```

Comment:
Did you run this through elm-format?

Reply:
I didn't. I just downloaded it and did it now. Thanks for looking at this and the suggestion!